index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./Assets/style/reset.css" />
+    <link rel="stylesheet" href="./Assets/style/style.css" />
+    <title>Apple</title>
+  </head>
+  <body>
+    <div class="grid-section">
+      <main class="grid-main">
+        <nav class="globalheader">
+          <ul>
+            <li><a class="link-one" href="#"></a></li>
+            <li><a class="link-two" href="#"></a></li>
+            <li><a class="link-three" href="#"></a></li>
+            <li><a class="link-four" href="#"></a></li>
+            <li><a class="link-five" href="#"></a></li>
+            <li><a class="link-six" href="#"></a></li>
+            <li><a class="link-seven" href="#"></a></li>
+            <li><a class="link-eight" href="#"></a></li>
+          </ul>
+          <div class="search-container">
+            <form action="#">
+              <input type="text" />
+            </form>
+            <span class="material-icons search-icon">
+              search
+            </span>
+          </div>
+        </nav>
+        <div class="main-content">
+          <h1>
+            <a href="#"
+              ><img
+                alt="What will your verse be?"
+                src="https://archive.fo/UW4oR/d7dd20c4365117e1cc1b3c2492508e9ba842e5d5.png"
+            /></a>
+          </h1>
+          <h2>
+            <a href="#"
+              ><img
+                alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story."
+                src="https://archive.fo/UW4oR/98e684c02dfa6d288709a583e375cfbdea4b3f7d.png"
+            /></a>
+          </h2>
+          <div>
+            <a href="#"></a>
+          </div>
+        </div>
+      </main>
+      <aside class="grid-sub-1"></aside>
+      <aside class="grid-sub-2"></aside>
+      <aside class="grid-sub-3"></aside>
+      <aside class="grid-sub-4"></aside>
```

Comment:
- [ ] These aside images need to be links also. Since you are using background images, in this case, you could add `a` tags inside each aside and give them full height and width.

Reply:
Thank you for your review. I will fix the way you explain but I wonder what if I wrap <aside> with <a> tag?

Reply:
Thank you for your review. I will fix the way you explain but I wonder what if I wrap side
with anchor tag?